src/components/PlanSelector.tsx

```diff
@@ -0,0 +1,39 @@
+import type React from "react";
+import type { Plan, PlansResponse } from "../utils/api";
+import PlanParameters from "./PlanParameters";
+import Select from "@mui/material/Select";
+import { useState } from "react";
+import { MenuItem } from "@mui/material";
+
+type PlanSelectorProps = {
+  planResponse: PlansResponse;
+};
+
+const PlanSelector: React.FC<PlanSelectorProps> = (
+  props: PlanSelectorProps,
+) => {
+  const [currentPlan, setCurrentPlan] = useState<Plan | undefined>(undefined);
+
+  return (
+    <div>
+      <h3>Available Plans</h3>
+      <Select
```

Comment:
Fine for now. Going forward we need to decide what this UI really is. If it is a generic plan runner (which it currently wants to be), then this is probably not the correct component (would not be very usable with 12 plans) - maybe a scrollable list view and to the right the form for the selected plan. However the UI for this project should probably mapping-like.